### PR TITLE
Adds placeholder for live stream and notice

### PR DESCRIPTION
**Pull request by schalkventer**
**Blocking**
- Link to live stream (implement in two locations)

image

### CODE REVIEW

File: _layouts/homepage.html [hunk 1]
```diff
@@ -24,15 +24,18 @@ <h1 class="Page-title">The South African National Budget {{ page.financial_year
     <div class="u-marginBottom25">
       {%
         include components/universal/Alert/index.html
-        title="Reminder"
-        message="The Medium Term Budget Policy Statement (MTBPS), a Cabinet policy statement tabled in Parliament about three months before the annual budget, will be streamed live from the vulekamali homepage on Wednesday, the 25th of October."
+        title="Please note"
+        message="...pending description"
```
Comment (longhotsummer) [inline _layouts/homepage.html]: Can this have links?

"The full Medium Term Budget Policy Statement is [available on National Treasury's website](URL). The MTBPS documents and data will be added to vulekamali.gov.za soon."

File: _data/videos.yaml [hunk 2]
```diff
@@ -3,6 +3,13 @@ description: "South Africa's National and Provincial budget data from National T
 selected_tab: learning-centre
 selected_sidebar: videos
 data:
+- id: mtbps
```
Comment (longhotsummer) [inline _data/videos.yaml]: Will this mean it will show up in the video pages, too?

Comment (longhotsummer) [reply _data/videos.yaml]: Another option is just to put the iframe into the page directly. That's what I initially had in mind, I hadn't thought of this mechanism.

Comment (schalkventer) [reply _data/videos.yaml]: Yes.

File: _layouts/homepage.html (hunk 1 above)
Comment (schalkventer) [reply _layouts/homepage.html]: Including the link inline adds a bit of complexity. Can I perhaps just put the actually URL as text in the message? Maybe in brackets?

File: _data/videos.yaml (hunk 2 above)
Comment (schalkventer) [reply _data/videos.yaml]: The iframe isn't responsive by default, so I'll have to add some CSS hacking to it (which is already done in the video component). Personally I don't see a problem with it being in the videos section.

How strongly do you feel about this?

Comment (longhotsummer) [reply _data/videos.yaml]: This is fine.

File: _layouts/homepage.html (hunk 1 above)
Comment (longhotsummer) [reply _layouts/homepage.html]: Please drop the include completely, and just use raw HTML.

File: _layouts/homepage.html [hunk 3]
```diff
@@ -21,12 +21,18 @@ <h1 class="Page-title">The South African National Budget {{ page.financial_year
...
+        {%
+          include components/universal/Icon/index.html
+          type="alert"
+          size="large"
+        %}
+        <span class="Alert-heading">
+          Please note
+        </span>
+      </div>
+      <div class="Alert-text">The full Medium Term Budget Policy Statement is <a href="#">available on National Treasury's website</a>. The MTBPS documents and data will be added to vulekamali.gov.za soon.</div>
```
Comment (longhotsummer) [inline _layouts/homepage.html]: Great. This link should be the following, if they stick to their old scheme:

`http://www.treasury.gov.za/documents/mtbps/2018/default.aspx`

**Comment by schalkventer** [comment]
image

**Pull request merged**